i18n/it/cosmic_settings.ftl

```diff
@@ -1,23 +1,23 @@
-app = Impostazioni COSMIC
+app = Impostazioni di COSMIC
 
 unknown = Sconosciuto
 
 number = { $number }
 
 ## Desktop
 
-desktop = Scrivania
+desktop = Desktop
```

Comment:
I chose Desktop because, while Scrivania is something that I saw my whole life in various linux-based distributions, it's just inconsistent.
You go in `/home` and see "Desktop"; every program expects to see `/home/Desktop` and not `/home/Scrivania`.

You speak about less tech-savvy users: Windows users are generally the most "less tech-savvy" users that you could find in the entire planet, they're used to "Desktop", not "Scrivania".

I think "Desktop" is the best of both worlds since it accomodates "the pro" with "the noob".